keepers/basic-keeper.ts

```diff
@@ -1,27 +1,30 @@
 import proxy from "../utils/arweave-proxy";
+import { Keeper, PriceDataSigned, PriceDataKeeped } from "../types";
 
-const VERSION = "0.005";
+const mockKeeper: Keeper = {
+  async keep(price: PriceDataSigned): Promise<PriceDataKeeped> {
+    // TODO: think about naming unification
```

Comment:
Yes, we should also add the "provider" field and we can rename "data-latest" to "data".
Instead of token let's use "symbols" as we will store multiple prices in a single tx.
I don't think source and value will be needed.
(We may add top level indexing for most popular tokens adding tags with name equal to token symbol and value equal to the price, like:
AR: 14.21,
ETH: 1455.3